### ISSUE TITLE
AssertionError on Ubuntu 18.04.2 LTS, Mackup 0.8.25, Python 3.6.7

### ISSUE THREAD

**Issue by alancwoo** [lra/mackup#1408]
I'm trying to `mackup restore` on a machine running

- Ubuntu 18.04.2 LTS
- Mackup 0.8.25
- Python 3.6.7

It fails immediately with the following:

```
Traceback (most recent call last):
  File "/home/REDACTED/.pyenv/versions/3.6.7/bin/mackup", line 10, in <module>
    sys.exit(main())
  File "/home/REDACTED/.pyenv/versions/3.6.7/lib/python3.6/site-packages/mackup/main.py", line 102, in main
    verbose)
  File "/home/REDACTED/.pyenv/versions/3.6.7/lib/python3.6/site-packages/mackup/application.py", line 26, in __init__
    assert isinstance(files, set)
AssertionError
```

I sync via dropbox, and to debug I made a tar.gz of the original mackup folder and copied/extracted it directly with no luck :( Not sure how to proceed to debug further, I've also tried `mackup restore -v` with no luck.

**Comment by lubocheng** [comment]
me too, it may be a spelling error, try to fix with this:
`sudo vi /usr/local/lib/python2.7/dist-packages/mackup/application.py`
go to line 26, note it or replace it with follow:
`assert isinstance(files, list)`
Then it will be ok

**Comment by dstroot** [comment]
@lubocheng Brilliant!  This worked.  Without this fix Mackup is plain broken on python 2 and 3

**Comment by ileitch** [comment]
Same here on macOS with Mackup 0.8.25

**Comment by misaka** [comment]
Duplicate of https://github.com/lra/mackup/issues/1410

Looks like the issue is caused by the change in this commit: https://github.com/lra/mackup/pull/1402/commits/54e5d98ac84e9db10039cd2b28fe5c02b4460541#diff-cfe235ad3bb75c756c89b6a752e6dd4aR142. It seems that adding `sorted` there turns the set into an array.

```python
(Pdb) self.apps[name]['configuration_files']
{'.mackup', '.mackup.cfg'}
(Pdb) sorted(self.apps[name]['configuration_files'])
['.mackup', '.mackup.cfg']
```

which fails the assertion in the stack trace above:

```python
assert isinstance(files, set)
```

**Comment by misaka** [comment]
PR available #1411
